config/webpack.config.dev.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  devtool: '#source-map',
+  externals: ['cozy'],
+  module: {
+    loaders: [{
+      test: require.resolve('cozy-bar/dist/cozy-bar.js'),
+      loader: 'imports?css=./cozy-bar.css'
+    }]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      __SERVER__: JSON.stringify('http://app.cozy.local'),
```

Comment:
Pretty good question, I thinks it's a perfectly valid way to use local assets in an app served locally. I never tested it, but it's the same process like the `volume` flag on the docker image :smiley: